elements/rh-menu-dropdown/demo/dropdown-with-group-headings.html

```diff
@@ -0,0 +1,38 @@
+<div id="menu-dropdown-container">
+  <rh-menu-dropdown text="Basic toggle" info-icon-name="auto-light-dark-mode">
+    <span slot='label'>Basic toggle</span>
+    <rh-menu>
+      <rh-menu-item-group group-heading="Group heading">
+        <rh-menu-item>Action <p slot='description'>This is a description.</p></rh-menu-item>
+        <rh-menu-item href="#">Link <p slot='description'>This is a description.</p></rh-menu-item>
+      </rh-menu-item-group>
+      <hr/>
+      <rh-menu-item-group group-heading="Group heading">
+        <rh-menu-item>Separated action</rh-menu-item>
+        <rh-menu-item href="#">Separated link</rh-menu-item>
+      </rrh-menu-item-group>
+    </rh-menu>
```

Comment:
Should these `rh-menu-item-groups` have different `group-heading` values?

I realize they are just presentational. But might just be good for the demo to have the represent a clear and defined seperate header and list pairing.